cypress/integration/rewards.spec.js

```diff
@@ -0,0 +1,299 @@
+engine.addProvider(
+  new RpcSubprovider({
+    rpcUrl: 'http://localhost:8545',
+  })
+)
+/*
+engine.on('block', function(block) {
+  console.log('================================')
+  console.log(
+    'BLOCK CHANGED:',
+    '#' + block.number.toString('hex'),
+    '0x' + block.hash.toString('hex')
+  )
+  console.log('================================')
+})
+*/
+engine.on('error', function(err) {
+  console.error(err.stack)
+})
+
+engine.start()
+
+const user1 = new PrivateKeyProvider(
+  'A8A54B2D8197BC0B19BB8A084031BE71835580A01E70A45A13BABD16C9BC1563',
+  'http://localhost:8545'
+)
+
+const user2 = new PrivateKeyProvider(
+  'ce8e3bda3b44269c147747a373646393b1504bfcbb73fc9564f5d753d8116608',
+  'http://localhost:8545'
+)
+
+web3A.setProvider(user1)
+web3B.setProvider(user2)
+
+
+context('Aragon', () => {
+  context('Rewards', () => {
+
+  before(() => {
+    cy.on('window:before:load', win => {
+      win.web3 = web3A
+      console.log('A', web3A.eth.accounts)
+      console.log('B', web3B.eth.accounts)
+          })
+    cy.visit(
+      'http://localhost:3000/#/0x5b6a3301a67A4bfda9D3a528CaD34cac6e7F8070'
+    )
+  })
+
+/////////////////////////////////////////////////////////////////////////////
+// create new merit reward
+/////////////////////////////////////////////////////////////////////////////
+
+    it('creates New Reward (Merit)', () => {
+      const tomorrow = addDays(new Date(), 3) + ''
+
+      cy.contains('span', 'Rewards', { timeout: 150000 })
+        .click()
+        .wait(4000)
+
+      cy.getButton('New Reward')
+        .click()
+        .wait(1000)
+        .appFind('input[name="description"]')
+        .type('Merit Reward')
+        .appFind('input[name="amount"]')
+        .type(3)
+        .appFind('div[data-e2e="reward-amount-currency"]')
+        .click('right')
+        .within($el => {
+          cy.contains('div', 'autark')
+            .click()
+        })
+
+      cy.appContains('aside', 'New Reward')
+        .appContains('div', 'Select a token')
+        .click()
+        .appFind('span[data-e2e-token="autark"]')
+        .click({ force: true })
+        .appFind('input[name="periodEnd"]')
+        .type(tomorrow)
+        .appFind('input[name="periodEnd"]')
+        .trigger('keydown', {
+          keyCode: 13,
+          which: 13,
+        })
+
+      cy.getButton('Submit Reward')
+        .click()
+        .wait(1000)
+
+      cy.contains('button', 'Create transaction').click()
+      cy.contains('button', 'Close')
+        .click()
+        .wait(3000)
+    })
+
+/////////////////////////////////////////////////////////////////////////////
+// create new dividend reward
+/////////////////////////////////////////////////////////////////////////////
+
+    it('creates New Reward (Dividend)', () => {
+      const tomorrow = addDays(new Date(), 98) + ''
+
+      cy.contains('span', 'Rewards', { timeout: 150000 })
+        .click()
+        .wait(4000)
+
+      cy.getButton('New Reward')
```

Comment:
Please convert the above wait into a timeout for thus function so we don't have to wait the full 4 seconds every time this loads